spec/support/shared_examples/data_page_shared_example.rb

```diff
@@ -0,0 +1,7 @@
+shared_examples 'a data page' do |tables|
+  it 'contains all requires tables' do
+    tables.each do |table_name|
+      expect page.has_table? table_name
```

Comment:
Upon further inspection, the two aren't interchangeable like that it seems. Doing as you did yields failed tests (doesn't find `data_table` on either). I can't seem to figure out *how* they're different though. I thought it might work on the parent element (nope) or maybe I had to include the `#` to denote it's looking at ID (also nope).